GVFS/GVFS.UnitTests.Windows/Windows/Upgrader/UpgradeOrchestratorUnitTests.cs

```diff
@@ -0,0 +1,168 @@
+﻿using GVFS.Common;
+using GVFS.Common.Git;
+using GVFS.Common.Tracing;
+using GVFS.Upgrader;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.IO;
+
+namespace GVFS.UnitTests.Upgrader
+{
+    [TestFixture]
+    public class UpgradeOrchestratorUnitTests
+    {
+        private const string OlderThanLocalVersion = "1.0.17000.1";
+        private const string LocalGVFSVersion = "1.0.18115.1";
+        private const string NewerThanLocalVersion = "1.1.18115.1";
```

Comment:
Should `NewerThanLocalVersion` be different than `LocalGVFSVersion`?

Reply:
that is a bit subtle... Maybe we can make the version numbers more distinct